packages/@sfx/sayt-driver-plugin/README.md

```diff
@@ -0,0 +1,46 @@
+# SF-X Sayt Driver Plugin
+
+This package contains the SF-X Sayt (Search-as-you-type) Driver plugin.
+
+## Prerequisites
+
+This plugin depends on the following plugins:
+
+- `dom_events`
+- `sayt`
+- `search`
+
+These plugins must be registered either before or in the same batch as
+this plugin.
+
+## Usage
+
+To use the plugin, simply instantiate it and register it with Core:
+
+```js
+const saytDriverPlugin = new SaytDriverPlugin({});
+core.register(saytDriverPlugin);
+```
+
+This plugin accepts the following options:
+* `productTransformer` (optional)
+    * A function that accepts a Record object and returns a transformed
+    product. This argument is only necessary if the View Layer is being
+    used. More generally, this function is an opportunity to transform
+    the data into a useable format for your application's front-end.
+
+## Events
+
+This plugin listens for and dispatches a number of events.
+
+### Received
+
+* `sfx::autocomplete_fetch_data`: When received, a request to the GroupBy Sayt API is made. An `sfx::autocomplete_received_results` event is dispatched with the results. In case of an error, the `sfx::autocomplete_sayt_error` event is dispatched instead.
```

Comment:
I believe we concluded that we should use the sfx-events constant names in READMEs and internal documentation, and link to sfx-events. If I recall correctly, we should have an example in one of the View layer packages.

Reply:
You're right. I was using an example that I think has changed since rebasing.